account/manager.go

```diff
@@ -1931,35 +2031,11 @@ func locateAccountInput(tx *wire.MsgTx, account *Account) (int, error) {
 	return 0, errors.New("account input not found")
 }
 
-// signInput signs a P2WKH or NP2WKH input of a transaction.
-func (m *manager) signInput(ctx context.Context, tx *wire.MsgTx,
-	in chanfunding.Coin, idx int,
-	sigHashType txscript.SigHashType) (*input.Script, error) {
-
-	signDesc := &lndclient.SignDescriptor{
-		Output: &wire.TxOut{
-			Value:    in.Value,
-			PkScript: in.PkScript,
-		},
-		HashType:   sigHashType,
-		InputIndex: idx,
-	}
-	inputScripts, err := m.cfg.Signer.ComputeInputScript(
-		ctx, tx, []*lndclient.SignDescriptor{signDesc},
-	)
-	if err != nil {
-		return nil, err
-	}
-
-	return inputScripts[0], nil
-}
-
-// signAccountInput signs the account input in the spending transaction of an
+// decorateAccountInput signs the account input in the spending transaction of an
 // account. If the account is being spent with cooperation of the auctioneer,
 // their signature will be required as well.
-func (m *manager) signAccountInput(ctx context.Context, tx *wire.MsgTx,
-	account *Account, idx int, sigHashType txscript.SigHashType) ([]byte,
-	[]byte, error) {
+func (m *manager) decorateAccountInput(account *Account, packet *psbt.Packet,
```

Comment:
`godoc` can be updated a bit: this doesn't aisng anylonger, but adds the pool specific PSBT information so it can sign properly.